_data/team/roles/non_core_library_maintenance.yml

```diff
@@ -0,0 +1,80 @@
+role: Non-core library maintenance
+description: |
+  The non-core library maintenance team is responsible for the
+  managing the maintenance of the all MDAnalysis packages
+  that are not the core MDAnalysis library.
+tasks:
+  - Managing issues and pull-requests for sub-projects
+  - Managing new features and updates
+  - Managing maintenance
+  - Working with CI and CD team and infrastructure
+subroles:
+
+  - subrole: distopia
+    current_leads: []
+    current_members: []
+    historical_members:
+      - Hugo MacDermott-Opeskin
+    
+
+  - subrole: cookiecutter-mdakit
+    current_leads:
+      - Lily Wang
+    current_members: []
+    historical_members:
+      - Lily Wang
+    
+
+  - subrole: mda-encore
+    current_leads: []
+    current_members: []
+    historical_members:
+      - Ian Kenney
+    
+
+  - subrole: hole2-mdakit
```

Comment:
We're missing PSA, and waterdynamics here.

Reply:
I can be lead for PSA; it came out of my group so at least that makes some sense.

@fiona-naughton can you adopt waterdynamics for now? (Think of it as a kitten that needs a new home...)

Reply:
Well when you put it like that 😸 But yes, happy to take this on

Reply:
Added!